packages/interactor/src/converge.ts

```diff
@@ -0,0 +1,22 @@
+const win: { performance?: unknown } = (typeof(window) === 'object') ? window : {};
+const performance = (typeof(win.performance) === 'object') ? win.performance : require('perf_hooks').performance;
```

Comment:
Is it possible to use a dynamic `import()` for this instead? Using `require()` here isn't a huge deal because it's supported by build tools like Webpack, but it is a node thing and I like to try to avoid using it in code that could be run in the browser. I'm not sure about the performance implications.

Reply:
The idea is that we shouldn't hit the `require` in the browser at all, since we want to use `window.performance` in the browser and only use the `perf_hooks` package in a node environment. So we would actually want this to *not* be packages. But this may not at all be the right way of achieving this.